data/registry/instrumentation-go-splunkpq.yml

```
@@ -14,3 +14,4 @@ authors:
 urls:
   repo: https://github.com/signalfx/splunk-otel-go/tree/main/instrumentation/github.com/lib/pq/splunkpq
 createdAt: 2022-01-20
+isFirstParty: true
```

```suggestion
isFirstParty: false
```

this is not a first party instrumentation